conda/recipes/dask-cuda/meta.yaml

```diff
@@ -29,8 +29,9 @@ requirements:
   host:
     - python
     - pip
-    - rapids-build-backend>=0.3.0,<0.4.0.dev0
-    - setuptools>=64.0.0
+    {% for r in data["build-system"]["requires"] %}
```

Comment:
Where is build-system defined? Is it read from https://github.com/rapidsai/dask-cuda/blob/afc27f4c8a7bdc73bc777f96d20b03c525b28a83/pyproject.toml#L1-L6 ?

Reply:
Yep exactly